pep-0394.txt

```diff
@@ -10,6 +10,7 @@ Author: Kerrick Staley <mail@kerrickstaley.com>,
         Carol Willing <willingc@gmail.com>,
 Status: Active
 Type: Informational
+Topic: Packaging
```

Comment:
I'd exclude this -- it's not related to Packaging tooling interoperability, but rather about having a clear entry point for Python on Unix-like systems.